docs/assets/demo.html

```diff
@@ -0,0 +1,86 @@
+<select id="demo-options" aria-label="demo options"></select>
+
+<pre id="demo-display" style="white-space: break-spaces;">
+    Choose an example to analyze
+</pre>
+
+<script>
+    const base = "https://pymwp1.uk.r.appspot.com/";
+    const displayPre = document.getElementById('demo-display');
+    const select = document.getElementById('demo-options');
+    const examples = {
```

Comment:
Ok, so we have to "hard-code" by hand the example we want to display.
Good to know.

Reply:
for now yes but I _could_ make it load the options from server. Still working on this.